docs/Reference/CLI/Subcommands/Migrate-Database.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Database migration subcommand options
+---
+
+# `migrate-database`
+
+[Migrate the database to a LevelDB2 database](../../../HowTo/Migrate-Database.md).
+
+!!! note
+
+    As of v21.5.0, new Teku installations use a LevelDB2 database. Previous Teku versions use
+    RocksDB which continue to be supported.
+
+## config-file
+
+=== "Syntax"
+
+    ```bash
+    --config-file=<FILE>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --config-file=/home/me/me_node/config.yaml
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_CONFIG_FILE=/home/me/me_node/config.yaml
+    ```
+
+The path to the [YAML configuration file](../../../HowTo/Configure/Use-Configuration-File.md).
+The default is `none`.
+
+## data-base-path, data-path
+
+=== "Syntax"
+
+    ```bash
+    --data-base-path=<PATH>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --data-base-path=/home/me/me_node
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_DATA_BASE_PATH=/home/me/me_node
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    data-base-path: "/home/me/me_node"
+    ```
+
+The path to the Teku data directory. The default directory is OS dependent:
+
+* macOS: `~/Library/teku`
+* Unix/Linux: `$XDG_DATA_HOME/teku` if `$XDG_DATA_HOME` is set; otherwise `~/.local/share/teku`
+* Windows: `%localappdata%\teku`.
+
+The default Docker image location is `/root/.local/share/teku`.
+
+## data-storage-archive-frequency
+
```

Comment:
archive frequency is completely ignored, basically the only options that will have any impact is the network, and the data location, I'm not sure even the storage mode would be a good thing to try to change... I haven't tried having an archive database and attempt to use a prune mode while running this subcommand...
We're basically doing a byte copy of data, which is why it equates to disk speed - there's no interpretation of anything in real terms, we're pouring data from 1 location to the next...